bitcoin/src/sign_message.rs

```diff
@@ -104,7 +104,7 @@ mod message_signing {
         pub fn serialize(&self) -> [u8; 65] {
             let (recid, raw) = self.signature.serialize_compact();
             let mut serialized = [0u8; 65];
-            serialized[0] = recid.to_i32() as u8 + if self.compressed { 31 } else { 27 };
+            serialized[0] = i32::from(recid) as u8 + if self.compressed { 31 } else { 27 };
```

Comment:
```suggestion
            // Adding recid.to_u8() tracked in: https://github.com/rust-bitcoin/rust-secp256k1/issues/774
            serialized[0] = i32::from(recid) as u8 + if self.compressed { 31 } else { 27 };
```

Reply:
`to_u8`? Shouldn't it be just an enum of 4 possible values?

Reply:
I think we don't have those yet. See #774.

Would be nice to slip them into a 0.30.1..

Reply:
You've linked something unrelated. But looking at docs.rs, we do have an enum already which means that simply `recid as u8` ought to work.

Reply:
Oops, wrong repo. I meant https://github.com/rust-bitcoin/rust-secp256k1/issues/774

But good call that we can at least drop the i32::from for now.

Reply:
`recid as u8` - gold.